posthog-node/src/posthog-node.ts

```diff
@@ -173,7 +173,9 @@ export class PostHog extends PostHogCoreStateless implements PostHogNodeV1 {
             additionalProperties[`$feature/${feature}`] = variant
           }
         }
-        const activeFlags = Object.keys(flags || {}).filter((flag) => flags?.[flag] !== false)
+        const activeFlags = Object.keys(flags || {})
+          .filter((flag) => flags?.[flag] !== false)
+          .sort()
```

Comment:
This makes the tests pass and not be flaky, but I think it's actually the right thing to do. Makes it easy to compare different calls in the activity log.